src/vm/eventpipe.h

```diff
@@ -387,8 +399,10 @@ class EventPipe
 
     static CrstStatic s_configCrst;
     static bool s_tracingInitialized;
-    static EventPipeConfiguration *s_pConfig;
-    static EventPipeSessions *s_pSessions;
+    static EventPipeConfiguration *s_pConfig; // Volatile?
```

Comment:
We should mark this as `Volatile`. This can get freed in `EventPipe::Shutdown()` concurrently while other threads are still accessing it during `WriteEventInternal`.

Reply:
I would leave it for a different PR.